clickstream/start.sh

```diff
@@ -4,9 +4,9 @@
 . ../utils/helper.sh
 
 check_env || exit 1
-check_running_cp 5.0 || exit 
-check_running_elasticsearch 5.6.5 || exit 1
-check_running_grafana 5.0.3 || exit 1
+check_running_cp 5.0.1 || exit 
+#check_running_elasticsearch 5.6.5 || exit 1
```

Comment:
Why is `check_running_elasticsearch` and `check_running_grafana` commented out?

Reply:
by mistake, reverted in https://github.com/confluentinc/examples/pull/85/commits/80292d5a7ae549f9eca39f7b363aaacb2e6da455